integration_tests/Main.php

```diff
@@ -120,6 +120,21 @@
     'googleapis/google/cloud/dialogflow/v2/dialogflow_v2.yaml',
     'googleapis/google/cloud/dialogflow/v2/dialogflow_grpc_service_config.json'
 )) ? $ok : false;
+// // TODO: googleapis/google/cloud/dialogflow/cx/v3/ has wrong capitalization somewhere, so monolith crashes
```

Comment:
Add a comment with a link or the specific failure case? 

Also FYI: For Java, the monolith did some other weird capitalization things we've had to special-case in the monolith. Not sure if this applies to PHP though. Cases that come to mind:
- "IAM" -> "Iam" but in general, FOOBar stays the same.
- 123abc becomes 123Abc. This was a legit bug because the monolith produced 123ABC and 123Abc in two different places (since fixed).

Reply:
Yes, I suspect you're right that there's some pre-generator munging that happens for this API, which isn't reproduced in this test.